src/v8/v8_script.rs

```diff
@@ -14,20 +14,25 @@ use crate::v8::v8_context_scope::V8ContextScope;
 use crate::v8::v8_value::V8LocalValue;
 
 /// JS script object
+#[derive(Debug)]
 pub struct V8LocalScript<'isolate_scope, 'isolate> {
     pub(crate) inner_script: *mut v8_local_script,
     pub(crate) isolate_scope: &'isolate_scope V8IsolateScope<'isolate>,
 }
 
+/// A persisted script is a JavaScript-compiled code, which isn't tied
+/// to the isolate it was compiled for. Hence it doesn't have any
+/// lifetime boundaries and can live on its own, and converted into a
+/// [V8LocalScript] later when it is required.
+#[derive(Debug)]
 pub struct V8PersistedScript {
```

Comment:
I think maybe its a good time to discuss the safety guarantees here, we will probably also need it for the inspector. Sure we can not verify on compile time that the persisted context do not outlive the isolate, but maybe we can have some run time check and get an error when transforming the persisted value back to local value if it was done with the wrong isolate?

Reply:
We should probably do the same for `V8PersistedValue`, but maybe lets keep it for another PR.